src/microbe_stage/Meteor.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using ThriveScriptsShared;
+
+/// <summary>
+///   Contains definitions of meteors for meteor impact event
+/// </summary>
+public class Meteor : IRegistryType
+{
+    public WorldEffectVisuals VisualEffect;
+
+    public List<string> Chunks = new();
+
+    public Dictionary<Compound, double> Compounds = new();
```

Comment:
```suggestion
    public Dictionary<Compound, float> Compounds = new();
```

In Thrive compound amounts are always stored as floats.